Makefile

```diff
@@ -455,12 +455,16 @@ test-build-js-native-api: all build-js-native-api-tests
 test-build-node-api: all build-node-api-tests
 
 .PHONY: test-all
-test-all: test-build ## Run everything in test/.
+test-all: test-build ## Run default tests with both Debug and Release builds.
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=debug,release
 
 test-all-valgrind: test-build
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=debug,release --valgrind
 
+.PHONY: test-all-suites
+test-all-suites: test-build test-js-native-api test-node-api | bench-addons-build ## Run all test suites.
+	$(PYTHON) tools/test.py $(PARALLEL_ARGS) --mode=$(BUILDTYPE_LOWER) test/*
```

Comment:
AFAICT We don't need the `test/*`

Reply:
[side note] Could we have a target that is not dependent on anything else?

Reply:
Without `test/*`, it runs 2413 test for me locally. With it, it runs 2639 tests. I think it's because leaving off `test/*` means that `test.py` will skip the suites in its `IGNORED_SUITES` list.